packages/studio-ui/src/web/views/FlowBuilder/sidePanelTopics/form/SaySomethingForm.tsx

```diff
@@ -52,13 +52,18 @@ const SaySomethingForm: FC<Props> = props => {
     }
   }, [props.currentFlowNode.id])
 
-  const renameNode = text => {
+  const onChange = text => {
```

Comment:
What's this class ? Why is this dupplicated ? 

Same comment goes for here, there's a lot of indentation